shared/p2p/service.go

```diff
@@ -159,20 +160,18 @@ func (s *Server) Subscribe(msg proto.Message, channel chan Message) event.Subscr
 
 // Send a message to a specific peer.
 func (s *Server) Send(msg proto.Message, peer Peer) {
-	// TODO
+	// TODO(175)
```

Comment:
do we need `#` in front of the numbers? I'm not sure if the regex is parsing for that

Reply:
The regex handles with or without the #. I think the hash is nice so i'll add it, but either is valid.